packages/common/package.json

```diff
@@ -89,6 +89,6 @@
     "lodash.get": "^4.4.2",
     "lodash.isempty": "^4.4.0",
     "xstate": "^5.18.2",
-    "zod": "^3.22.4"
```

Comment:
New version introduces string.date used in this PR - if common and workflow-core don't use the same version the types break.